client/web/src/components/ActivationDropdown/ActivationDropdown.story.tsx

```diff
@@ -64,20 +63,35 @@ export default config
 
 export const Loading: Story = () => <ActivationDropdown {...commonProps} activation={baseActivation()} />
 
-export const _04Completed: Story = () => (
+export const _04Completed: Story = args => (
     <ActivationDropdown
         {...commonProps}
         activation={{
             ...baseActivation(),
             completed: {
-                ConnectedCodeHost: boolean('ConnectedCodeHost', false),
-                DidSearch: boolean('DidSearch', false),
-                FoundReferences: boolean('FoundReferences', false),
-                EnabledRepository: boolean('EnabledRepository', false),
+                ...args,
             },
         }}
     />
 )
+_04Completed.argTypes = {
+    ConnectedCodeHost: {
+        control: { type: 'boolean' },
+        defaultValue: false,
+    },
+    DidSearch: {
+        control: { type: 'boolean' },
+        defaultValue: false,
+    },
+    FoundReferences: {
+        control: { type: 'boolean' },
+        defaultValue: false,
+    },
+    EnabledRepository: {
```

Comment:
```suggestion
    EnabledSharing: {
```

This isn't an error with your changes, but I noticed that `EnabledRepository` is no longer used in the UI, and instead the final step is now `EnabledSharing`. This affects the other story in this file, as well. Could either update it with these changes, or do a follow-up PR after this gets merged (to not have to deal with any merge conflicts now 😄).